File: src/main/java/com/revature/model/Guest.java
package com.revature.model;

import com.revature.dao.UserDao;

public class Guest extends User
{
    public Guest()
    {
    }

    public Guest(int userId, String firstName, String lastName, String email, String username, String password)
    {
        super(userId, firstName, lastName, email, username, password);
    }



    @Override
    public String toString()
    {
        return "Guest{} " + super.toString();
    }
}

File: src/main/webapp/js/guest.js
/**
 * Populates a reservation table row
 * @param reservation
 */
function populateGuestsTableRow(guest){
    $('#guests').append(
        `<tr>
            <td>${guest.userId}</td>
            <td>${guest.firstName} ${guest.lastName}</td>
            <td>${guest.email}</td>
        </tr>`
    );
}
